openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -59,7 +59,7 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                 scored = 'scored' if subsection.get('scored', False) else ''
                 graded = 'graded' if subsection.get('graded') else ''
                 %>
-                % if not subsection.get('complete', True) and subsection.get('due', timezone.now() + timedelta(1)) < timezone.now() and not reset_deadlines_banner_displayed:
+                % if subsection.get('due', timezone.now() + timedelta(1)) < timezone.now() and not subsection.get('complete', True) and not reset_deadlines_banner_displayed:
```

Comment:
It might help future readability if you use `timedelta(days=1)`. I had to go hit up the docs to double check that that's what the first argument is.

I think this whole line might be clearer if you pushed some of the calculations into the `<% %>` python block above, so that the conditional is just combining conditions, rather than combining and evaluating. That would also let you just do something like: 
```
due_date = subsection.get('due')
overdue = due_date is not None and due_date < timezone.now()
```
which might be clearer than the current setup.